app/src/pages/playground/PlaygroundOutput.tsx

```diff
@@ -15,25 +20,61 @@ import {
   PlaygroundOutputSubscription$data,
   PlaygroundOutputSubscription$variables,
 } from "./__generated__/PlaygroundOutputSubscription.graphql";
+import { isChatMessages } from "./playgroundUtils";
 import { PlaygroundInstanceProps } from "./types";
 
 interface PlaygroundOutputProps extends PlaygroundInstanceProps {}
 
+export function PlaygroundOutputMessage({ message }: { message: ChatMessage }) {
```

Comment:
not sure if we want to show this card in all cases but wanted it for span parsing, seems to be a common way to display

Reply:
I wouldn't export.

What do you mean you wanted it for "span parsing" ?

Reply:
oh yeah export was a mistake

also the parsing wording was a brain fart haha, wanted it for displaying output messages parsed from the original span** is what i meant to say